rc/ansi.kak

```diff
@@ -67,7 +67,11 @@ define-command \
     set-option buffer ansi_color_ranges %val{timestamp}
 }
 
-hook -group ansi global BufCreate '\*stdin(?:-\d+)?\*' %{
+define-command \
+    -docstring %{ansi-enable: render buffer and enable rendering new fifo data for current buffer.} \
+    -params 0 \
+    ansi-enable %{
+    ansi-render
```

Comment:
I like this because up to now I have been too lazy to write this myself. 

The lack of an `ansi-disable` command seems acceptable since this is mostly used for `*make*` - the user can just recreate a fresh `*make*` buffer.

Reply:
Yeah but now that you point it out, maybe I really should add an `ansi-disable`, for completeness sake.

Actually now that I think about it I can't see many uses for the `ansi-render` and `ansi-clear` commands alone. `ansi-enable` and `ansi-disable` would have the same results and it wouldn't conflict with anything, they just have the extra benefit that they work on fifos too.

Maybe we should consider replacing the original commands altogether?

Reply:
`ansi-enable; ansi-render` seems usually better than just `ansi-render`. However, on a very large buffer, `ansi-render` can take a considerable amount of time. So maybe there is still merit in doing it manually,like your current version does. Probably not a strong use case though; I think either way would work.